src/components/input/style.css

```diff
@@ -34,7 +34,7 @@ label {
 	padding: 0.6em 0.2em 0.6em 0.2em;
 }
 input {
-	padding: 1em 0.2em 0.2em 0.2em;
+	padding: 1.2em 0.3em 0.2em 0.4em;
```

Comment:
The padding needs fit with the padding for `:host:not([show-label]) input` So that they will take up the same amount of space next to each other.

Here is a screenshot from how it looks in our app if I use the new padding:
![image](https://user-images.githubusercontent.com/14332757/168540717-a1dba45e-f1c2-4bbf-90d3-0a2e6d3ca659.png)

Before it was 
`0.6em 0.2em 0.6em 0.2em` and `1em 0.2em 0.2em 0.2em` for the two states 
total padding width: `0.6+0.6 = 1+0.2`
And padding height: `0.2+0.2 = 0.2+0.2`

You can change both of them if you want to, but they need to have the same total padding height and width.